src/utils/rest-api.js

```diff
@@ -486,12 +486,13 @@ export function fetchTwoWindingsTransformers(
     currentNodeUuid,
     substationsIds
 ) {
-    return fetchEquipments(
+    return fetchNetworkElementsInfos(
         studyUuid,
         currentNodeUuid,
         substationsIds,
-        'Two windings transformers',
-        '2-windings-transformers'
+        EQUIPMENT_TYPES.TWO_WINDINGS_TRANSFORMER.type,
+        EQUIPMENT_INFOS_TYPES.TAB.type,
+        true
```

Comment:
why do all "inUpstreamBuiltParentNode" are now passed as "true" ? They were passed as "false" before if I'm not wrong